.circleci/config.yml

```diff
@@ -10,7 +10,9 @@ jobs:
       - node/with-cache:
           steps:
             - run: npm install
+            - run: npm run-script build
             - run: npm test
+            - run: npm run-script lint
```

Comment:
I would do `npm run build` and `npm run lint`

Reply:
Done.